packages/yarnpkg-builder/sources/commands/build/plugin.ts

```diff
@@ -108,18 +117,20 @@ export default class BuildPluginCommand extends Command {
             // get evaluated right now - until after we give it a custom require
             // function that will be able to fetch the dynamic modules.
             {apply: (compiler: webpack.Compiler) => {
-              compiler.hooks.compilation.tap(`MyPlugin`, (compilation: webpack.Compilation) => {
-                compilation.hooks.optimizeChunkAssets.tap(`MyPlugin`, (chunks: Set<webpack.Chunk>) => {
+              compiler.hooks.compilation.tap(`WrapperPlugin`, (compilation: webpack.Compilation) => {
+                compilation.hooks.optimizeChunkAssets.tap(`WrapperPlugin`, (chunks: Set<webpack.Chunk>) => {
                   for (const chunk of chunks) {
                     for (const file of chunk.files) {
                       // @ts-expect-error
-                      compilation.assets[file] = new webpack.sources.RawSource(
+                      compilation.assets[file] = new ConcatSource(
```

Comment:
```suggestion
                      compilation.assets[file] = new webpack.sources.ConcatSource(
```

Don't need to add the dependency for this as webpack already exports it

Reply:
> Don't need to add the dependency for this as webpack already exports it.

Ah ok - will remove.

> We should probably just support users customizing their configs, as there might be other, specific stuff, that people might need

Agreed.